packages/content-management/table_list_view_table/src/table_list_view_table.tsx

```diff
@@ -457,6 +458,8 @@ function TableListViewTableComp<T extends UserContentCommonSchema>({
         return item.references.find(({ id: refId }) => refId === _id) as SavedObjectsReference;
       });
 
+      const isReadonly = contentEditor.isReadonly || !itemIsEditable?.(item);
```

Comment:
I don't think `!itemIsEditable?.(item)` check is right. This will return true when `itemIsEditable` is not provided. Items should not be readonly when itemIsEditable is not provided.

Reply:
Good catch, fixed this in 702984450081